app/filtering.js

```diff
@@ -471,7 +471,7 @@ function registerPermissionHandler (session, partition) {
         response.push(false)
       } else if (permission === 'fullscreen' &&
         // The Torrent Viewer extension is always allowed to show fullscreen media
-        origin.startsWith('chrome-extension://' + config.torrentExtensionId)) {
+        mainFrameOrigin.startsWith('chrome-extension://' + config.torrentExtensionId)) {
```

Comment:
this throws an error when you go to http://www.orimi.com/pdf-test.pdf and click the fullscreen button

should be `requestingOrigin` instead of mainFrameOrigin i think

Reply:
actually it shouldn't - https://github.com/brave/browser-laptop/pull/13475/files#diff-8a9bac13ef6264feb2b6da7c18d86b3cR440 set the mainFrameOrigin to null deliberately

so i think this just has to check if mainFrameOrigin is null